aca-docker/bin/env_variables_all.txt

```diff
@@ -13,3 +13,8 @@ export CCD_BEFTA_SOLICITOR_4_PWD=Pa55word11
 export ES_ENABLED_DOCKER=true
```

Comment:
Delete this if it is no longer required

Reply:
Good spot - removed in https://github.com/hmcts/aac-manage-case-assignment/pull/191/commits/1839013564d73fe8371d1af428ccf78dfadfc25a